### PR TITLE
add Connectable interface

### PR DESCRIPTION
**Pull request by jandanielcz**

### CODE REVIEW

**Comment by jandanielcz** [comment]
GitHub API communication error. Please contact a human. (commit-ID: 0f5e9dcd96da420d9551ec7d6e581f64d7e72fdf).

File: lib/Connectable.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+interface Connectable
+{
+    /** 
+     * @return string
+     */
+    public function id();
+}
```
Comment (SytstemsTealake) [inline lib/Connectable.php:9]: :no_entry_sign: **Error**: File must end with a newline character (*Generic.Files.EndFileNewline.NotFound*).

File: lib/Connectable.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+interface Connectable
+{
+    /** 
+     * @return string
+     */
+    public function id();
```
Comment (SytstemsTealake) [inline lib/Connectable.php:8]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: lib/Connectable.php [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+interface Connectable
+{
+    /** 
+     * @return string
+     */
```
Comment (SytstemsTealake) [inline lib/Connectable.php:7]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: lib/Connectable.php [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+interface Connectable
+{
+    /** 
+     * @return string
```
Comment (SytstemsTealake) [inline lib/Connectable.php:6]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

File: lib/Connectable.php [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+interface Connectable
+{
+    /**
```
Comment (SytstemsTealake) [inline lib/Connectable.php:5]: :no_entry_sign: **Error**: Whitespace found at end of line (*Squiz.WhiteSpace.SuperfluousWhitespace.EndLine*).

Comment (SytstemsTealake) [inline lib/Connectable.php:5]: :no_entry_sign: **Error**: Missing short description in doc comment (*Generic.Commenting.DocComment.MissingShort*).

Comment (SytstemsTealake) [inline lib/Connectable.php:5]: :no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).

**Review by SytstemsTealake — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 10 errors 

***

**hashes-api**-scanning skipped

File: lib/Connectable.php [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+interface Connectable
+{
```
Comment (SytstemsTealake) [inline lib/Connectable.php:4]: :no_entry_sign: **Error**: Opening brace should be on the same line as the declaration for interface Connectable (*Generic.Classes.OpeningBraceSameLine.BraceOnNewLine*).

File: lib/Connectable.php [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<?php
```
Comment (SytstemsTealake) [inline lib/Connectable.php:1]: :no_entry_sign: **Error**: Missing file doc comment (*Squiz.Commenting.FileComment.Missing*).

Comment (SytstemsTealake) [inline lib/Connectable.php:1]: :no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-l1akxv.php, but found phpcs-scan-l1akxV.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).

Comment (SytstemsTealake) [inline lib/Connectable.php:1]: :no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-lwhmkz.php, but found phpcs-scan-lWhmKZ.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).

**Review by SytstemsTealake — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 1 error 

***

**hashes-api**-scanning skipped

**Review by SytstemsTealake — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 1 error 

***

**hashes-api**-scanning skipped

File: lib/Connectable.php (hunk 7 above)
Comment (SytstemsTealake) [inline lib/Connectable.php:1]: :no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-y5zk7w.php, but found phpcs-scan-Y5zk7w.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).

**Review by SytstemsTealake — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 1 error 

***

**hashes-api**-scanning skipped

File: lib/Connectable.php (hunk 7 above)
Comment (SytstemsTealake) [inline lib/Connectable.php:1]: :no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-pwaa9l.php, but found phpcs-scan-pwAa9l.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).

**Pull request merged**